docs/content/latest/deploy/kubernetes/single-zone/gke/helm-chart.md

```diff
@@ -45,9 +45,9 @@ You must have a GKE cluster that has Helm configured. If you have not installed
 
 The YugabyteDB Helm Chart has been tested with the following software versions:
 
-- GKE running Kubernetes 1.18 (or later) with nodes such that a total of 12 CPU cores and 45 GB RAM can be allocated to YugabyteDB. This can be three nodes with 4 CPU core and 15 GB RAM allocated to YugabyteDB. `n1-standard-8` is the minimum instance type that meets these criteria.
+- GKE running Kubernetes 1.18 (or later). The helm chart you will use here to install yugabytedb requires 3 master servers and 3 t-servers each with 2 cpus - total 12 cpus. Thus you will need a kubernetes cluster with more than 12 cpus. With standard 3 nodes cluster, you will need each node with 8 cpus. 
```

Comment:
Reworded a bit, but I'm still not clear about what the last sentence (about standard nodes) is trying to say.

Also pls verify that changing "requires" to "creates" is correct here!

```suggestion
- GKE running Kubernetes 1.18 (or later). The helm chart you use to install YugabyteDB creates 3 master servers and 3 tablet servers, each with 2 CPU cores, for a total of 12 CPU cores. This mean you need a Kubernetes cluster with more than 12 CPU cores. With a standard 3-node cluster, each node requires 8 CPU cores. 
```

Reply:
Hi @polarweasel  - I over-wrote on your suggestion - also changed from "This mean" to "This means". And changed the last sentence. Let me know if it looks good ? Thanks for your suggestions again!